configs/system/base.js

```diff
@@ -38,6 +38,19 @@ module.exports = {
         image: 'dashpay/sentinel:1.5.0',
       },
     },
+    tor: {
+      enable: true,
+      docker: {
+        image: 'dperson/torproxy',
+      },
+      proxy: {
+        port: 9050,
+      },
+      control: {
+        port: 9051,
+        password: 'torcontrolpassword',
```

Comment:
Not sure it's a good idea to have a default value. Maybe generate it on setup?

Reply:
We use a default value for dashd as well. Let's create a separate issue for randomizing passwords on startup?

Reply:
Sounds good!